subsys/dfu/dfu_target/src/dfu_target_modem_delta.c

```diff
@@ -12,7 +12,7 @@
 #include <logging/log.h>
 #include <dfu/dfu_target.h>
 
-LOG_MODULE_REGISTER(dfu_target_modem, CONFIG_DFU_TARGET_LOG_LEVEL);
+LOG_MODULE_REGISTER(dfu_target_modem_delta, CONFIG_DFU_TARGET_LOG_LEVEL);
```

Comment:
Maybe we should consider grouping all of them under just 1 `dfu_target` log name?

Reply:
I will add a task for it. IMO its not related to these changes.